Automattic/wp-calypso
# ci(build): Drop fallback build

#### Background

See #53222
To be merged _after_ #55775 and #56029

#### Changes proposed in this Pull Request

* Drop scripts used to build the fallback build. Now only the evergreen build can be built.

#### Testing instructions

Checkout this branch and verify the usual commands in your devflow still work (`yarn start`, `yarn build`... )
